.github/workflows/mirror-images.yml

```diff
@@ -12,9 +12,19 @@ on:
         type: string
         required: true
         default: "1.0.0"
+  workflow_run:
```

Comment:
This triggers a run of this workflow upon a completed run of 'Create Production Tag'.